packages/mantine/src/components/header/Header.tsx

```diff
@@ -23,7 +23,7 @@ export interface HeaderProps extends DefaultProps {
     /**
      * The tooltip text shown when hovering over the doc link icon
      */
-    docLinkTooltipLabel?: string;
+    docLinkTooltipLabel: string;
```

Comment:
This is a breaking change. Also, not all headers have a doc link, so making this prop required wouldn't be appropriate for everyone

Reply:
You are right this would be a breaking change for any code that uses Header component, the reason why I introduced this here is because in Tooltip, the `label` prop is required, and if nothing is passed, it will show a black icon. I reverted my change.